ci-operator/step-registry/gcp/provision/xpn-min-perm-cco-manual-serviceaccount/gcp-provision-xpn-min-perm-cco-manual-serviceaccount-ref.yaml

```
@@ -0,0 +1,17 @@
+ref:
+  as: gcp-provision-xpn-min-perm-cco-manual-serviceaccount
+  from_image:
+    namespace: ocp
+    name: "4.15"
```

Use the latest stable?
```suggestion
    name: "4.19"
```